.github/CODEOWNERS

```diff
@@ -36,15 +36,56 @@
 # directory in the root of your repository.
 #/docs/ @doctocat
 
-##### Jenkins #####
-/pkg/jenkins/ @CCFenner
-
-##### Sonar #####
-/pkg/sonar/ @CCFenner
-/resources/metadata/sonarExecuteScan.yaml @CCFenner
-/vars/sonarExecuteScan.groovy @CCFenner
-/cmd/sonarExecuteScan.go @CCFenner
-/cmd/sonarExecuteScan_test.go @CCFenner
-
-##### Orchestrator #####
-pkg/orchestrator/ @inf2381
+
+
+#####################
+# Default           #
+#####################
+
+*                                               @SAP/jenkins-library-admin
```

Comment:
need to use jenkins-library-team here I guess
Those are the colleagues typically working and approving

Reply:
But I'm also part of the jenkins-library-team team. I think it should be a team only containing the piper "core developers".